src/Equinor.ProCoSys.IPO.WebApi/Controllers/Invitation/InvitationsController.cs

```diff
@@ -48,13 +49,16 @@ namespace Equinor.ProCoSys.IPO.WebApi.Controllers.Invitation
     [Route("Invitations")]
     public class InvitationsController : ControllerBase
     {
+        private readonly ILogger<InvitationsController> _logger;
```

Comment:
why keep the logger here now!! Undo all changes in in this file. The file should not appear as changed in PR